classes/LocalizationPack.php

```diff
@@ -469,7 +509,7 @@ protected function installConfiguration($xml)
      * @param SimpleXMLElement $xml
      * @return bool
      */
-    protected function _installGroups($xml)
+    protected function installGroups($xml)
```

Comment:
Please do not rename protected and public functions without aliasing methods with old names to the new ones so that we can deprecate the old ones smoothly.